css-typed-om/Overview.bs

```diff
@@ -2743,7 +2743,9 @@ objects, the {{CSSURLImageValue/url}} attribute contains the URL that references
     when called,
     perform the following steps:
 
-    1. Return a new {{CSSURLImageValue}}
+    1. If the |url| passed into the constructor is not a [[url#valid-url-string]],
```

Comment:
I couldn't figure out how to link the URL spec correctly sorry :/

Reply:
That doesn't really work. You need to use the URL parser (`<a>URL parser</a>`) and make sure the return value is not failure. However, you'll also need to pass a base URL for that. And it seems you want to treat fragments as a special value per issue discussion so this probably needs some more thought.